LICENSE

```diff
@@ -1,3 +1,5 @@
+ https://github.com/WorldHealthOrganization/app
```

Comment:
This space is intentional - only way to sort it to the top of flutter's license list.